cmd/cluster/support/post.go

```diff
@@ -79,34 +79,24 @@ The cluster has a second failing ingress controller, which is not supported and
 		Args:              cobra.NoArgs,
 		DisableAutoGenTag: true,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			if err := p.Run(p.ClusterID); err != nil {
-				return fmt.Errorf("error posting limited support reason: %w", err)
```

Comment:
Extraneous error handling, this would log twice on top of `cobra`s builtin handling of failures in a `RunE` func def.